discord/ui/select.py

```diff
@@ -180,13 +206,35 @@ def max_values(self, value: int):
             raise ValueError("max_values must be between 1 and 25")
         self._underlying.max_values = int(value)
 
+    @property
+    def disabled(self) -> bool:
+        """:class:`bool`: Whether the select is disabled or not."""
+        return self._underlying.disabled
+
+    @disabled.setter
+    def disabled(self, value: bool):
+        self._underlying.disabled = bool(value)
+
+    @property
+    def channel_types(self) -> list[ChannelType]:
+        """List[:class:`discord.ChannelType`]: A list of channel types that can be selected in this menu."""
+        return self._underlying.channel_types
+
+    @channel_types.setter
+    def channel_types(self, value: list[ChannelType]):
+        if self._underlying.type is not ComponentType.channel_select:
+            raise InvalidArgument("channel_types can only be set on channel selects")
+        self._underlying.disabled = bool(value)
```

Comment:
If I had to guess, This should be setting channel_types
```suggestion
        self._underlying.channel_types = value
```

Reply:
welp

Reply:
I must've been half asleep writing this

Reply:
daily mood